libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
In cmsis of RTX offered in IAR compiler, Cortex-M is supported but Cortex-A is not supported.
Therefor, we have ported cmsis(for Cortex-A) of RTX for IAR that we have corresponded on their own to mbed environment.
We reflected its change to core_caFunc.h(573-814) and deleted the following comment.
`#error IAR Compiler support not implemented for Cortex-A.`